tests/tools/oecertdump/host/host.cpp

```diff
@@ -0,0 +1,466 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <limits.h>
+#include <openenclave/host.h>
+#include <openenclave/internal/report.h>
+#include <openenclave/internal/sgxcertextensions.h>
+#include <openenclave/internal/tests.h>
+#include <openssl/bio.h>
+#include <openssl/pem.h>
+#include <openssl/x509.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "oecertdump_u.h"
+
+#if defined(__linux__)
+#include <dlfcn.h>
+#endif
+
+#include "../../../../common/sgx/quote.h"
```

Comment:
Should not have SGX specific code in a non-SGX-specific file.
Please move any SGX specific code to a filename or directory that has sgx in the name, but leave any non-SGX-specific code in this file.

Reply:
Added sgx_quote.cpp and h.